### PR TITLE
Home Screen Initial Mockup + Updated Color Palette

### PR DESCRIPTION
**Pull request by hanskamin**
- Created an initial mockup of our home screen.
- Added the Y's official color palette to our app-wide theming in place of the generic Ionic color palette.

### CODE REVIEW

File: capacitor.config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "appId": "io.ionic.starter",
+  "appName": "ymun-app",
```
Comment (ChrisBlair2019) [inline capacitor.config.json]: Does this file need to be included?

File: ionic.config.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
   "name": "ymun-app",
-  "integrations": {},
+  "integrations": {
```
Comment (ChrisBlair2019) [inline ionic.config.json]: Does this file need to be included?

File: ios/.gitignore [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# NPM renames .gitignore to .npmignore
+# In order to prevent that, we remove the initial "."
+# And the CLI then renames it
+
+App/build
+App/Pods
```
Comment (ChrisBlair2019) [inline ios/.gitignore]: Does this file need to be included?

File: ios/App/App/capacitor.config.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "appId": "io.ionic.starter",
+  "appName": "ymun-app",
+  "bundledWebRuntime": false,
```
Comment (ChrisBlair2019) [inline ios/App/App/capacitor.config.json]: Does this file need to be included?

File: ios/App/App/config.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+  <access origin="*" />
```
Comment (ChrisBlair2019) [inline ios/App/App/config.xml]: Does this file need to be included?

File: ios/App/Podfile [hunk 6]
```diff
@@ -0,0 +1,20 @@
+platform :ios, '11.0'
+use_frameworks!
+
+# workaround to avoid Xcode 10 caching of Pods that requires
+# Product -> Clean Build Folder after new Cordova plugins installed
+# Requires CocoaPods 1.6 or newer
+install! 'cocoapods', :disable_input_output_paths => true
+
```
Comment (ChrisBlair2019) [inline ios/App/Podfile]: Does this file need to be included?

**Comment by hanskamin** [comment]
@ChrisBlair2019 none of those files need to be included, I could've sworn I added them to the gitignore but apparently not lol

**Pull request merged**